### ISSUE TITLE
Option to allow for auto-resuming to happen instantly after hovering over subtitle

### ISSUE THREAD

**Issue by mathiaslovnes** [killergerbah/asbplayer#498]
As of now, if I hover over the subtitle, I either have to hit space to continue playing, or enable the auto-resume feature, which takes way too long to come into play. I'd like an option to allow for the player to resume instantly after I remove my mouse from the subtitle line

**Comment by killergerbah** [comment]
> which` takes way too long to come into play

Can you go into more details about this problem? What website are you seeing this on?
There's nothing about the current implementation that would specifically cause auto-resume to be slow.

**Comment by mathiaslovnes** [comment]
It is for me, at least. If I enable "Auto-pause when mousing over subtitles" and I hover over the subtitle, it does pause immediately. However, I have to move my mouse away from the subtitle and wait like 3 seconds for it to resume again. 

This is just a minor issue, though. Appreciate the great work you've done. It's been fantastic for sentence-mining!

**Comment by killergerbah** [comment]
Got it, can you link me to the website where youre seeing this? Just so I have a way to see the issue for myself and possibly come up with a workaround.

**Comment by killergerbah** [comment]
I don't believe this is actionable as the delay for the video to resume is out asbplayer control.
